src/test-gui/java/de/qabel/desktop/ui/LayoutGuiTest.java

```diff
@@ -32,6 +35,15 @@ private LayoutController createController() {
     }
 
     @Test
+    public void showAboutViewOnStart() {
+        clientConfiguration.onSelectIdentity(this::aboutViewIsShown);
```

Comment:
It's unclear what's the test here.
`clientConfiguration.onSelectIdentity(.... do stuff that should fail if code is wrong ...)`

The identity is not set inside the test so I would guess it's set during the setup ... which is already completed when the tests starts. Thus, if the `onSelectIdentity` does not call the new observer instantly when we already have an identity (which is the behavior I would expect), then `this::aboutViewIsShown` is never called. Plausible test could be
```JAVA
@Test
public void showAboutViewOnStartIfAnIdentityExists() {
    new AboutPage(...).waitFor(); // or this.waitForAboutView()
}
```